{
  try {
    handleClient();
  }
 catch (  Exception e) {
    log("Exception during handleClient in the TestWebServer: " + e.getMessage());
  }
  log(this + " terminated");
}
